MIMWebClient/Core/Helpers.cs

```diff
@@ -29,5 +29,16 @@ public int dice(int number, int minSize, int maxSize)
 
             return sum;
         }
+
+        
+        /// <summary>
+        /// Returns a random string.
+        /// </summary>
+        /// <param name="possible"></param>
+        /// <returns></returns>
+        public static String Random(params String[] possible)

```

Comment:
Awesome, thanks. I think the name should be GetRandomString. To make it clear it's not a random number or something else random. Thanks